src/js/service/ImportService.js

```diff
@@ -28,20 +28,22 @@
   /**
    * Given an image object and some options, create a new Piskel and open it
    * for editing.
-   * @param {!Image} image
-   * @param {!Object} options
-   * @param {!string} options.importType - 'single' if not spritesheet
-   * @param {!number} options.frameSizeX
-   * @param {!number} options.frameSizeY
-   * @param {number} [options.frameOffsetX] only used in spritesheet imports.
-   * @param {number} [options.frameOffsetY] only used in spritesheet imports.
-   * @param {!boolean} options.smoothing
-   * @param {number} [options.frameRate] in frames per second
-   * @param {function} [onComplete]
+   * @param {Image} image
+   * @param {Object} options
+   *        - {String}  importType 'single' if not spritesheet
+   *        - {String}  name
+   *        - {Boolean} smoothing
+   *        - {Number}  frameSizeX
+   *        - {Number}  frameSizeY
+   *        - {Number}  frameOffsetX (only used in spritesheet imports)
+   *        - {Number}  frameOffsetY (only used in spritesheet imports)
```

Comment:
We lost the documentation for `options.frameRate`, which is one of our customizations, in the merge here.  Looks like the functionality is correctly preserved below.